src/TinyCiv.Shared/Events/Client/StartGameClientEvent.cs

```diff
@@ -0,0 +1,6 @@
+namespace TinyCiv.Shared.Events.Client;
+
+/// <summary>
+/// Use to trigger game start
+/// </summary>
+public record StartGameClientEvent : ClientEvent;
```

Comment:
@Randwarf @jshimas when you get event from ```ListenForGameStartReady``` you can use this event to trigger game start

after triggering you should receive stuff from ```ListenForGameStart```